From 239f813d8b0b19ef30391829b57670b3b5220142 Mon Sep 17 00:00:00 2001
From: qjohn <townex22@gmail.com>
Date: Sat, 16 Jun 2018 15:05:21 +0700
Subject: [PATCH 03/25] msm8953-common: Kill dexpreopt

Change-Id: Ifda9d4392252a461284501e3d12569a82933f8c6
---
 BoardConfigCommon.mk | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index 5960fde..3582f23 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -103,16 +103,6 @@ ENABLE_CPUSETS := true
 # Crypto
 TARGET_HW_DISK_ENCRYPTION := true
 
-# Dexpreopt
-ifeq ($(HOST_OS),linux)
-  ifneq ($(TARGET_BUILD_VARIANT),eng)
-    ifeq ($(WITH_DEXPREOPT),)
-      WITH_DEXPREOPT := true
-    endif
-  endif
-endif
-WITH_DEXPREOPT_BOOT_IMG_ONLY ?= true
-
 # Display
 MAX_VIRTUAL_DISPLAY_DIMENSION := 4096
 TARGET_FORCE_HWC_FOR_VIRTUAL_DISPLAYS := true
-- 
2.7.4

